------------------------------------------------------------
revno: 14176
revision-id: squid3@treenet.co.nz-20170701073514-uzowexcysowqostf
parent: squid3@treenet.co.nz-20170629125627-socq6szqysvm9ifa
fixes bug: http://bugs.squid-cache.org/show_bug.cgi?id=4687
author: Lubos Uhliarik <luhliari@redhat.com>
committer: Amos Jeffries <squid3@treenet.co.nz>
branch nick: 3.5
timestamp: Sat 2017-07-01 19:35:14 +1200
message:
  Bug 4687: Wrong names of components in man page, section SEE ALSO
------------------------------------------------------------
# Bazaar merge directive format 2 (Bazaar 0.90)
# revision_id: squid3@treenet.co.nz-20170701073514-uzowexcysowqostf
# target_branch: http://bzr.squid-cache.org/bzr/squid3/3.5
# testament_sha1: 9099c98de3cb8fc125dd9952373de42c079b0ccc
# timestamp: 2017-07-01 07:45:05 +0000
# source_branch: http://bzr.squid-cache.org/bzr/squid3/3.5
# base_revision_id: squid3@treenet.co.nz-20170629125627-\
#   socq6szqysvm9ifa
# 
# Begin patch
=== modified file 'src/squid.8.in'
--- src/squid.8.in	2017-01-01 00:16:45 +0000
+++ src/squid.8.in	2017-07-01 07:35:14 +0000
@@ -265,11 +265,11 @@
 .SH SEE ALSO
 .if !'po4a'hide' .B cachemgr.cgi "(8), "
 .if !'po4a'hide' .B squidclient "(1), "
-.if !'po4a'hide' .B pam_auth "(8), "
-.if !'po4a'hide' .B squid_ldap_auth "(8), "
-.if !'po4a'hide' .B squid_ldap_group "(8), "
+.if !'po4a'hide' .B basic_pam_auth "(8), "
+.if !'po4a'hide' .B basic_ldap_auth "(8), "
+.if !'po4a'hide' .B ext_ldap_group_acl "(8), "
 .if !'po4a'hide' .B ext_session_acl "(8), "
-.if !'po4a'hide' .B squid_unix_group "(8), "
+.if !'po4a'hide' .B ext_unix_group_acl "(8), "
 .br
 The Squid FAQ wiki
 .if !'po4a'hide' http://wiki.squid-cache.org/SquidFaq

